scene/surface-placement/src/main/java/com/esri/samples/surface_placement/SurfacePlacementController.java

```diff
@@ -63,73 +63,41 @@ public void initialize() {
       surface.getElevationSources().add(elevationSource);
       scene.setBaseSurface(surface);
 
-      // add a scene layer
+      // add buildings to the scene with a scene layer URI
       ArcGISSceneLayer sceneLayer = new
         ArcGISSceneLayer("http://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/Buildings_Brest/SceneServer/layers/0");
```

Comment:
this is also a bit too far over our line length guidance (over by a little bit is ok though). Suggest the following new line to break it up:

```java

      // add buildings to the scene with a scene layer URI
      ArcGISSceneLayer sceneLayer = new ArcGISSceneLayer(
        "http://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/Buildings_Brest/SceneServer/layers/0");
```